parsec/api/data/base.py

```diff
@@ -175,7 +176,7 @@ def verify_and_load(
         author_verify_key: VerifyKey,
         expected_author: Optional[DeviceID],
         expected_timestamp: Optional[DateTime] = None,
-        **kwargs,
+        **kwargs: Any,
```

Comment:
```suggestion
        **kwargs: Dict[str, Any],
```
`**kwargs` can be typed as a `dict` right ?

Reply:
nop, `*args` and `**kwargs` should be typed as the type of the parameter, not the type of the entire `args/kwargs` (this is because args is always a List[Any] and kwargs as Dict[str, Any]

Reply:
ok